File: lab_2/lab_2.py
def main():
    number = int(input("กรุณาระบุตัวเลขที่จะตรวจสอบ: "))
    print(f"เลข {number} เป็นเลข", end="")
    if number % 2 == 0:
        print("คู่")
    else:
        print("คี่")

if __name__ == "__main__":
    main()

File: lab_2/lab_2.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    int number;
    cout << "กรุณาระบุตัวเลขที่จะตรวจสอบ: ";
    cin >> number;
    cout << "เลข " << number << " เป็นเลข"; ;
    if (number % 2 == 0) {
        cout << "คู่" << endl;
    } else {
        cout << "คี่" << endl;
    }

    return 0;
}
